<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorldPage</class>
 <widget class="QWidget" name="WorldPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>615</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="toolTip">
      <string/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_Website_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Visit HTH Partners and Donate Today!!!!!!!!</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: solid;
border-width: 2px;
border-color: #34bcaa;
border-radius: 10px;
font-size: 11px;</string>
      </property>
      <property name="text">
       <string>Partners</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Website_1">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>110</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Visit Help The Homeless Coin Website</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: solid;
border-width: 2px;
border-color: #34bcaa;
border-radius: 10px;
font-size: 11px;</string>
      </property>
      <property name="text">
       <string>HTH Coin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Website">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Visit Help The Homeless Worldwide A NJ Nonprofit Corporation</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: solid;
border-width: 2px;
border-color: #34bcaa;
border-radius: 10px;
font-size: 11px;</string>
      </property>
      <property name="text">
       <string>Worldwide</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Website_3">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>400</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Visit Help The Homeless Coin Explorer and Track Your HTH Coins</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: solid;
border-width: 2px;
border-color: #34bcaa;
border-radius: 10px;
font-size: 11px;</string>
      </property>
      <property name="text">
       <string>Explorer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Website_2">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>110</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Trade Help The Homeless Coin Today!!!</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: solid;
border-width: 2px;
border-color: #34bcaa;
border-radius: 10px;
font-size: 11px;</string>
      </property>
      <property name="text">
       <string>Buy HTH</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Website_5">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>230</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Check out the latest HTH News!!!</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: solid;
border-width: 2px;
border-color: #34bcaa;
border-radius: 10px;
font-size: 11px;</string>
      </property>
      <property name="text">
       <string>HTH News</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Mine">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>510</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Mine Help The Homeless Coin with Nvidia!</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: solid;
border-width: 2px;
border-color: #34bcaa;
border-radius: 10px;
font-size: 11px;</string>
      </property>
      <property name="text">
       <string>Nvidia</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Mine_AMD">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>510</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Mine Help The Homeless Coin with AMD!</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: solid;
border-width: 2px;
border-color: #34bcaa;
border-radius: 10px;
font-size: 11px;</string>
      </property>
      <property name="text">
       <string>AMD</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMine">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>340</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Mine Help The Homeless Coin!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelMineWarning">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>460</y>
        <width>391</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 15px; color:rgb(255, 15, 15); </string>
      </property>
      <property name="text">
       <string>Warning! You Must Place BAT file in Folder with QT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelNonprofit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nonprofit Charity Organizataion&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelHTHCoin">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;HTH Cryptourrency Info&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelHTHInfo">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>40</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;HTH Trading Info&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Website_6">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>170</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>HTHB is trading now!! More HTH Tokens to launch soon!!</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: solid;
border-width: 2px;
border-color: #34bcaa;
border-radius: 10px;
font-size: 11px;</string>
      </property>
      <property name="text">
       <string>HTH Tokens</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Website_7">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Visit HTH Chapters to see where you can make a difference!!!!</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: solid;
border-width: 2px;
border-color: #34bcaa;
border-radius: 10px;
font-size: 11px;</string>
      </property>
      <property name="text">
       <string>Chapters</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Website_8">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>400</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Visit Help The Homeless Coin Mining Pools</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: solid;
border-width: 2px;
border-color: #34bcaa;
border-radius: 10px;
font-size: 11px;</string>
      </property>
      <property name="text">
       <string>Pools</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Website_9">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>170</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Trade Help The Homeless on Binance Smart Chain Today!!!</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: solid;
border-width: 2px;
border-color: #34bcaa;
border-radius: 10px;
font-size: 11px;</string>
      </property>
      <property name="text">
       <string>Buy HTHB</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Website_10">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>230</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Visit Coin Gecko for value of Help The Homeless Coin</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: solid;
border-width: 2px;
border-color: #34bcaa;
border-radius: 10px;
font-size: 11px;</string>
      </property>
      <property name="text">
       <string>CoinGecko</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
